_posts/2022-10-22-101-post-kickstart.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Robots 101 - Kickstarted, now what?
+---
+
+Welcome to the start of Robot 101, our series where we guide you through the competition and give some advice on your next steps.
+
+So, you attended our kickstart event and are now ready to build your robot! But where should you start? We’re here to help™!
+
+We’ve found that most teams split up the different parts of the robot between them. It’s a good idea to meet up and discuss which tasks each person wants to accomplish and give each other roles inside the team. Of course, these can change throughout the time between kickstart and competition, but make sure everyone has a job to do. It’s important to make sure there's always at least two or three people that can do each task type. Don’t forget tasks outside the robot itself either, like project management, running the social media, or theming.
+
+Take some time together to plan your strategy for the game: how are you going to tackle the problem? What equipment would you want to experiment with? Remember to consider all areas of the robot - design, code, and mechanics. For some ideas, have a look on our [YouTube channel][youtube] to see some robots from previous competitions.. 
+
+Each member should also spend some time getting to know the docs on our website. This contains important information about the game’s rules, an introduction to your kit, as well as tips on how to program your robot. Your team supervisor will have been sent your invite link and password for our [Discord Server][discord-docs] - this is the best place to ask questions to the Blueshirts. Don’t be afraid to ask for help! Our volunteers have years of experience and are there to help you. You can also get your team supervisor to contact us through our email if you prefer. 
+
+Remember, you’re not just aiming to win First, Second, or Third! There are plenty of other awards in place for you to achieve throughout the year. For example, our “Teams on the Web” runs from now until the end of the competition - make sure you’re posting your progress on social media to be in the running! Check out the Rules for a full list of all the awards on offer.
+
+As part of your team, you’ll have a trusted adult who is acting as supervisor - their job is to guide you through the process of building your robot, not build it for you! We have a separate blog post on the role of a supervisor - you can read that [here][one-oh-one-supervisor]!

```

Comment:
For accessibility (and general friendliness) reasons we should aim to avoid using "here" as the link text. How about:
```suggestion
As part of your team, you’ll have a trusted adult who is acting as supervisor - their job is to guide you through the process of building your robot, not build it for you! We have a separate [blog post on the role of a supervisor][one-oh-one-supervisor].
```